libpcsxcore/psxinterpreter.c

```diff
@@ -217,14 +213,12 @@ static u32 INT_ATTR fetchICache(psxRegisters *regs, u8 **memRLUT, u32 pc)
 
 		if (((entry->tag ^ pc) & 0xfffffff0) != 0 || pc < entry->tag)
 		{
-			const u8 *base = memRLUT[pc >> 16];
```

Comment:
This should be `~0x0f` since it's fetching a cacheline. Please test with "ICache emulation" enabled.